scripts/rook.sh

```diff
@@ -8,6 +8,22 @@ ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 KUBECTL_RETRY=5
 KUBECTL_RETRY_DELAY=10
 
+trap log_errors ERR
```

Comment:
Now you need to use `errtrace` (`-E`) in addition to `errexit` (`-e`) to properly trap `ERR`.

Reply:
Not sure, with a small test script this seems to work just fine?
```bash
#!/bin/bash -e

trap log_exit ERR

function log_exit() {
	set -x
	echo "oh no"
}

false

exit 0
```

Reply:
I think it works because `false` is not in a function. If you have a function that fails this won't work see: https://stackoverflow.com/questions/35800082/how-to-trap-err-when-using-set-e-in-bash

Reply:
Oh, ok! Yes, failing inside a function indeed makes a difference. Will adapt it with `-E` and exit when the logging function gets called (otherwise it runs 2x, it seems).